src/window_skill.cpp

```diff
@@ -86,8 +86,8 @@ void Window_Skill::DrawItem(int index) {
 
 		std::stringstream ss;
 		ss << costs;
-		contents->TextDraw(rect.x + rect.width - 28, rect.y, color, "-");
-		contents->TextDraw(rect.x + rect.width - 6, rect.y, color, ss.str(), Text::AlignRight);
+		contents->TextDraw(rect.x + rect.width - 24, rect.y, color, "-");
+		contents->TextDraw(rect.x + rect.width, rect.y, color, ss.str(), Text::AlignRight);
```

Comment:
RPG_RT uses printf, we should do the same for compatibility and to cleanup the code.

Change this to a single TextDraw call with `fmt::format("-{:3d}", costs)`

Reply:
Changed.